### PR TITLE
feat(dev/SprintGraph) : made sprint progress graph

### PR DESCRIPTION
**Pull request by Pendyala-thanvi**
### Functionality:
Sprint progress graph

### Solution:
The progress width increases with increase in time

### Risk level:
- [ ] high 
- [x] medium
- [ ] low

### How to test:
ng serve
firebase emulators:start

### CODE REVIEW

**Review by arjund-06 — APPROVED** [review]
Ship it

**Comment by viv9k** [comment]
I need demo on this.

File: src/app/body/board/sprint-details/sprint-details.component.css [hunk 1]
```diff
@@ -28,4 +28,19 @@
...
     background-color: var(--primary-color);
+}
+
+#Progressgraph {
+    height: 10px; 
+    margin-top: 10px;
+    margin-bottom: 20px;
+    margin-left: 20px;
+    margin-right: 20px;
+    border-radius: 5px;
```
Comment (viv9k) [inline src/app/body/board/sprint-details/sprint-details.component.css:39]: Too much of static details, try reducing it as much as possible

**Review by viv9k — CHANGES_REQUESTED** [review]
Please do the required changes

**Review by viv9k — APPROVED** [review]
Ship it

**Pull request merged**